js/src/alert.js

```diff
@@ -133,12 +133,11 @@ const Alert = (($) => {
 
     static _jQueryInterface(config) {
       return this.each(function () {
-        let $element = $(this)
-        let data     = $element.data(DATA_KEY)
+        let data = $(this).data(DATA_KEY)
```

Comment:
This doesn't seem like an improvement to me. We shouldn't waste cycles reconstructing the same jQuery object multiple times.